labels.go

```diff
@@ -102,6 +103,7 @@ func CountLabel(s string) (labels int) {
 
 // Split splits a name s into its label indexes.
 // www.miek.nl. returns []int{0, 4, 9}, www.miek.nl also returns []int{0, 4, 9}.
+// .www.miek.nl. returns []int{0, 1, 5, 10}
```

Comment:
hmmm, this might be a bug, because the domain name is not valid.... good also call it garbage in, garbage out.